docs/sidebars.js

```diff
@@ -0,0 +1,23 @@
+const sidebars = {
+  sidebar: [
+    {
+      type: "category",
+      label: "ZIO Logging",
+      collapsed: false,
+      link: { type: "doc", id: "index" },
+      items: [
+        'logger-setup-in-zio-application',
+        'formatting-log-records',
+        'logger-context-and-annotations',
+        'json-console-logger',
+        'colorful-console-logger',
```

Comment:
i think we should merge
colorful-console-logger, json-console-logger to one page

console-logger
and structure it in similar way like jpl or slf4j page

have libraryDependencies section, general description, examples ...
to have some consistency in pages

Reply:
I only merged them. Let's leave the extra works you have mentioned on further PRs.